crawler/dockerutils.py

```diff
@@ -349,7 +349,7 @@ def get_docker_storage_driver():
 
     # Step 3, we default to "devicemapper" (last resort)
 
-    if driver not in ('btrfs', 'devicemapper', 'aufs'):
+    if driver not in ('btrfs', 'devicemapper', 'aufs', 'vfs'):
```

Comment:
if it is possible to do all this vfs stuff in a separate commit & PR, it would be better to do that.